src/lib/learn-client/schemas.ts

```diff
@@ -120,15 +120,5 @@ export const ProductPageSchema = Joi.object({
 			)
 			.required(),
 		showProductSitemap: Joi.boolean(),
-		docsCta: Joi.object({
```

Comment:
🤔 We should check whether this is still accepted in the learn api content sync repo. There could be an issue if an author _tries_ to use this, the sync succeeds and then we get an error in building the app.

Reply:
Perhaps its just an additional cleanup task for removing it in the content sync 🧼

Reply:
Ah great call! 🙌  I've opened https://github.com/hashicorp/learn-api-content-sync/pull/68 to clean this up in the `learn-api-content-sync` repo.